### PR TITLE
Added spanish translation

### PR DESCRIPTION
**Pull request by kant**

### CODE REVIEW

File: src/langES.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+*/
+
+
+export default {
+toolbox: {
```
Comment (Ovilia) [inline src/langES.js]: Indent here is not correct.

Comment (kant) [reply src/langES.js]: @Ovilia: You are right. The javascript must be end with this 3 final strings:

```
        }
    }
};
```

Comment (Ovilia) [reply src/langES.js]: @kant Please correct the indent and push again. Other things look right.

Comment (kant) [reply src/langES.js]: @Ovilia Indent fixed

**Pull request closed without merging**